lib/api/deploy.js

```diff
@@ -38,7 +38,7 @@ const EXT = 'zip'
 const HANDLER = 'handler'
 const pMkdir = util.promisify(temp.mkdir)
 
-const getProjectPath = (target) => path.join(target, 'project')
+const getProjectPath = target => path.join(target, 'project')
```

Comment:
You have prettier problems, make sure you have done `npm i` and restarted VS Code.

Reply:
Isn't this how it should be formatting it?

Reply:
nope